mail_layout_force/models/email_layout_mapping.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class EmailLayoutMapping(models.Model):
+    _name = "email.layout.mapping"
+
+    layout_id = fields.Many2one(
+        "ir.ui.view",
+        domain=[("type", "=", "qweb")],
+        required=True,
+        help="Select a layout that this layout will be substituted.",
+    )
+    substitute_layout_id = fields.Many2one(
+        "ir.ui.view",
+        domain=[("type", "=", "qweb")],
+        required=True,
+        help="Select a layout that this layout will substitute.",
+    )
+    models = fields.Many2many("ir.model")
```

Comment:
```suggestion
    model_ids = fields.Many2many("ir.model")
```

Please add help text as well.